spatialdata/_core/_spatialdata_ops.py

```diff
@@ -281,3 +288,165 @@ def _describe_paths(paths: list[list[Union[int, str]]]) -> str:
             transformations.append(g[path[i]][path[i + 1]]["transformation"])
         sequence = Sequence(transformations)
         return sequence
+
+
+##
+def concatenate_tables(tables: list[AnnData]) -> Optional[AnnData]:
```

Comment:
this I'm also not super sure on its use. My understanding is that it provides a function for the user to concatenate tables __outside__ of SpatialData yet compliant with SpatialData constraints. Why would a user do that instead of concatenating the SpatialData directly (which btw could just be SpatialData objs with only table) ? 
Furthermore, if the user really wants to do this, then the validator would spot errors (maybe a check for nans should be added) and the user would have to fix it this anyway.

In general, I don't see very much how much the user should handle elements outside of spatialdata, and hence I don't see why we should be providing functions for this kind of usage. Maybe worth discussing briefly @scverse/spatialdata ?

Reply:
You are correct, this function is supposed to be private. It will only be called by `concatenate(sdatas)` and not by the user. I will change it to private.